pkg/cortex/cortex.go

```diff
@@ -88,7 +88,7 @@ func (c *Config) RegisterFlags(f *flag.FlagSet) {
 	c.Server.MetricsNamespace = "cortex"
 	c.Target = All
 	c.Server.ExcludeRequestInLog = true
-	f.Var(&c.Target, "target", "target module (default All)")
+	f.Var(&c.Target, "target", "The Cortex service to run. Supported values are: all, distributor, ingester, querier, query-frontend, table-manager, ruler, alertmanager, configs.")
```

Comment:
Default value is now missing.

Reply:
We don't need to specify the default value in the description, because it's picked from the default value set for the flag (few lines above).

The `-help` is:

```
  -target value
        The Cortex service to run. Supported values are: all, distributor, ingester, querier, query-frontend, table-manager, ruler, alertmanager, configs. (default all)
```

While the related config doc is:

```
# The Cortex service to run. Supported values are: all, distributor, ingester,
# querier, query-frontend, table-manager, ruler, alertmanager, configs.
# CLI flag: -target
[target: <string> | default = "all"]
```

Reply:
Great, good to know. Thanks!